File: using c/queue/queueimplement.c
#include<stdio.h>
int front=0;
int rear=0;
int n=10;
int a[10];
int push(int);
int pop();
int display();
int main()
{
    push(10);
    push(20);
    push(30);
    push(40);
    push(50);
    push(60);
    push(70);
    push(80);
    push(90);
    push(100);
    pop();
    pop();
    display();
}
int push(int element){
    if(rear==n){
        printf("\noverflow condition\n");
    }
    else{
        rear++;
        a[rear]=element;
        printf("%d ",element);
    }
    display();
}
int pop(){
    if(front==rear){
        printf("\nunderflow condition\n");
    }
    else{
            int x;
            while(x<rear){
                a[x]=a[x+1];
                x++;
            }
            rear=rear-1;    
    }
    display();
}
int display(){
    if(rear==0){
        printf("queue is empty\n");
    }
    else{
        int i;
        printf("\n");
        for(i=front+1;i<=rear;i++){
            printf("%d ",a[i]);
        }
    }
}

File: using python/queue/queueimplement.py
class Queue:
    def __init__(self):
        self.Queue=list()
        self.front=0
        self.rear=0
    def insert(self,element):
            self.Queue.append(element)
            self.rear+=1
            print(self.Queue)
    def remove(self):
        if(self.rear != self.front):
            del self.Queue[0]
            self.rear-= 1
            print(self.Queue)
        else:
            print("Queue is empty")

    def sizeOfQueue(self):
        return len(self.Queue)
    
x=Queue()
x.insert(2)
x.insert(3) 
x.insert(5)
x.insert(6)
x.insert(8)
x.sizeOfQueue()
x.remove()
x.remove()
x.remove()
x.sizeOfQueue()       